index.js

```diff
@@ -62,6 +64,11 @@ controller.hears('invite', 'direct_mention', (bot, message) => {
 controller.on('team_join', onboard);
 
 /**
+ * Invited guests
+ */
+controller.hears(['mis parceros', 'invitados'], 'direct_message', guests);
```

Comment:
how about just `parceros` on the `hears` so it's not a multi-word command?

you could also add multiple words, `llaverias`, `parceros`, `neas`, `ñeros` that way its more complete